rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java

```diff
@@ -30,25 +30,48 @@
 import java.util.List;
 
 import static co.rsk.peg.PegTestUtils.createFederation;
+import static co.rsk.peg.PegTestUtils.createP2shErpFederation;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 class PegUtilsTest {
     private static final BridgeConstants bridgeMainnetConstants = BridgeMainNetConstants.getInstance();
     private static final NetworkParameters btcMainnetParams = bridgeMainnetConstants.getBtcParams();
     private static final Context context = new Context(bridgeMainnetConstants.getBtcParams());
-    ActivationConfig.ForBlock activations = ActivationConfigsForTest.tbd600().forBlock(0);
-
-    private BridgeStorageProvider provider;
+    private static final ActivationConfig.ForBlock activations = ActivationConfigsForTest.tbd600().forBlock(0);
 
     private static final int FIRST_OUTPUT_INDEX = 0;
     private static final int FIRST_INPUT_INDEX = 0;
-    private Address unknownAddress;
+
+    private BridgeStorageProvider provider;
+    private Address userAddress;
+
+    private Federation retiredFed;
+    private List<BtcECKey> retiringFedSigners;
+    private Federation retiringFederation;
+    private List<BtcECKey> activeFedSigners;
+    private Federation activeFederation;
+    private Wallet liveFederationsWallet;
+
 
     @BeforeEach
     void init() {
         provider = mock(BridgeStorageProvider.class);
-        unknownAddress = BitcoinTestUtils.createP2PKHAddress(btcMainnetParams, "unknown");
+        userAddress = BitcoinTestUtils.createP2PKHAddress(btcMainnetParams, "userAddress");
+
+        retiredFed = bridgeMainnetConstants.getGenesisFederation();
+
+        retiringFedSigners = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+            new String[]{"fa01", "fa02", "fa03"}, true
+        );
+        retiringFederation = createFederation(bridgeMainnetConstants, retiringFedSigners);
+
+        activeFedSigners = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+            new String[]{"fa01", "fa02", "fa03", "fa04", "fa05"}, true
+        );
+        activeFederation = createP2shErpFederation(bridgeMainnetConstants, activations, activeFedSigners);
+
+        liveFederationsWallet = new BridgeBtcWallet(context, Arrays.asList(activeFederation, retiringFederation));
```

Comment:
I think we should set this up for each test. Not sure it's the best idea to have active and retiring federation always by default

Reply:
Done